src/ServicePulse.Host/vue/src/components/configuration/HealthCheckNotifications.vue

```diff
@@ -243,4 +243,148 @@ onMounted(() => {
   </template>
 </template>
 
-<style></style>
+<style>
+.screen-intro {
+  margin: 30px 0;
+}
+
+.onoffswitch {
+    -moz-user-select: none;
+    -ms-user-select: none;
+    -webkit-user-select: none;
+    user-select: none;
+    position: relative;
+    width: 76px;
+}
+
+.onoffswitch-checkbox {
+  display: none;
+}
+
+.onoffswitch-label {
+  border: 2px solid #929E9E;
+  border-radius: 20px;
+  cursor: pointer;
+  display: block;
+  margin-top: 8px;
+  overflow: hidden;
+}
+
+.onoffswitch-inner {
+  display: block;
+  margin-left: -100%;
+  transition: margin 0.3s ease-in 0s;
+  width: 200%;
+}
+
+.onoffswitch-inner:before, .onoffswitch-inner:after {
+  box-sizing: border-box;
+  color: white;
+  display: block;
+  float: left;
+  font-family: Trebuchet, Arial, sans-serif;
+  font-size: 14px;
+  font-weight: bold;
+  height: 30px;
+  line-height: 30px;
+  padding: 0;
+  width: 50%;
+}
+
+.onoffswitch-inner:before {
+  background-color: #00A3C4;
+  color: #FFFFFF;
+  content: "ON";
+  padding-left: 10px;
+}
+
+.onoffswitch-inner:after {
+  background-color: #F2F6F7;
+  color: #929E9E;
+  content: "OFF";
+  padding-right: 10px;
+  text-align: right;
+}
+
+.onoffswitch-switch {
+  background: #FFFFFF;
+  border: 2px solid #929E9E;
+  border-radius: 20px;
+  bottom: 0;
+  display: block;
+  margin: 6px;
+  position: absolute;
+  right: 42px;
+  top: 0;
+  transition: all 0.3s ease-in 0s;
+  width: 22px;
+}
+
+.onoffswitch-checkbox:checked + .onoffswitch-label .onoffswitch-inner {
+  margin-left: 0;
+}
+
+.onoffswitch-checkbox:checked + .onoffswitch-label .onoffswitch-switch {
+  right: 0px;
+}
+
+.box-header {
+  padding-bottom: 3px;
+  padding-top: 2px;
+}
+
+.box-header ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+.lead {
+  -ms-word-wrap: break-word;
+  word-wrap: break-word;
+  color: #181919 !important;
+  font-size: 14px !important;
+  font-weight: bold;
+  margin-bottom: 3px;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+
+p.endpoint-metadata {
+  display: inline-block;
+  margin-top: 4px;
+  padding-right: 30px;
+}
+
+.endpoint-metadata sp-moment {
+  padding-top: 0;
+}
```

Comment:
I'm not sure if this is needed since I believe sp-moment is replace in Vue by TimeSince component.